### PR TITLE
fix(curriculum): Update hint text for step 61 of calorie counter project

### LINKED ISSUE #53499: Update hint text for step 61 of calorie counter project
### Describe the Issue

We have this post on the forum
https://forum.freecodecamp.org/t/learn-form-validation-by-building-a-calorie-counter-step-61/668041

The camper was trying to pass step 61 with this code 

```js
function getCaloriesFromInputs(list) {
  let calories = 0;

  for (let i = 0; i < list.length; i++) {
    const currVal = cleanInputString(list[i].value);
    const invalidInputMatch = isInvalidInput(currVal);

    if (invalidInputMatch) {
      alert(`Invalid Input: ${invalidInputMatch[0].value}`);
    }
  }
}
```
The hint text was telling them this
```md
You should use template literal syntax to display the first value in the `invalidInputMatch` array after the `Invalid Input: ` text.
```

We should update the wording to make it more clear that they should get the first element of that `invalidInputMatch` array.

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-form-validation-by-building-a-calorie-counter/step-61

### Your code

see explanation above

### Expected behavior

see explanation above

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Nitz2611**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #53499

### CODE REVIEW

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9bc53735149084390e5d0.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ You should use a template literal to pass the `Invalid Input: ` message to the `
 assert.match(code.split(/function\s+getCaloriesFromInputs/)[1], /alert\(`Invalid Input: /);
 ```
 
-You should use template literal syntax to display the first value in the `invalidInputMatch` array after the `Invalid Input: ` text.
+You should display the first element in the `invalidInputMatch` array after `Invalid Input:`  text using template literal.
```
Comment (JoyShaheb) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9bc53735149084390e5d0.md:29]:
```suggestion
You should display the first element in the `invalidInputMatch` array after `Invalid Input: `  text using template literal.
```

**Review by JoyShaheb — CHANGES_REQUESTED** [review]
just a minor change

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9bc53735149084390e5d0.md (hunk 1 above)
Comment (Nitz2611) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9bc53735149084390e5d0.md]: done

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9bc53735149084390e5d0.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ You should use a template literal to pass the `Invalid Input: ` message to the `
 assert.match(code.split(/function\s+getCaloriesFromInputs/)[1], /alert\(`Invalid Input: /);
 ```
 
-You should use template literal syntax to display the first value in the `invalidInputMatch` array after the `Invalid Input: ` text.
+You should display the first element in the `invalidInputMatch` array after `Invalid Input: ` text using template literal.
```
Comment (Sembauke) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-form-validation-by-building-a-calorie-counter/63c9bc53735149084390e5d0.md:29]:
```suggestion
You should display the first element of the `invalidInputMatch` array after the `Invalid Input:` text by using a template literal.
```

**Review by Sembauke — CHANGES_REQUESTED** [review]

**Review by Sembauke — APPROVED** [review]

**Review by JoyShaheb — APPROVED** [review]
👍

**Pull request merged**